### PR TITLE
Paladin/Gunbreaker buff downtime tracking

### PR DESCRIPTION
**Pull request by qaphla**
This PR adds the PLD self-buffs Fight or Flight and Requiescat, the GNB self-buff No Mercy, and the GNB resource-generator Bloodfest to the list of oGCDs that xivanalysis tracks the downtime for.

### CODE REVIEW

File: src/parser/jobs/pld/modules/OGCDDowntime.tsx [hunk 1]
```diff
@@ -1,9 +1,28 @@
 import ACTIONS from 'data/ACTIONS'
 import CooldownDowntime from 'parser/core/modules/CooldownDowntime'
 
+const ALLOWEDDOWNTIMEFOF = 4000
+const ALLOWEDDOWNTIMEREQ = 4000
+
+const FIRSTUSEOFFSETFOF = 24000
```
Comment (rachellhea) [inline src/parser/jobs/pld/modules/OGCDDowntime.tsx]: Probably worth sticking a comment here describing why we allow first use offset to be so big.

File: src/parser/jobs/gnb/modules/OGCDDowntime.tsx [hunk 2]
```diff
@@ -1,9 +1,20 @@
 import ACTIONS from 'data/ACTIONS'
 import CooldownDowntime from 'parser/core/modules/CooldownDowntime'
 
+const FIRSTUSEOFFSETNOMERCY = 10000
```
Comment (Toastdeib) [inline src/parser/jobs/gnb/modules/OGCDDowntime.tsx]: These should be split with underscores for readability (e.g. `FIRST_USE_OFFSET_NO_MERCY` and `FIRST_USE_OFFSET_BLOODFEST`. Same thing goes for the consts in the PLD module as well.

File: src/parser/jobs/pld/modules/OGCDDowntime.tsx (hunk 1 above)
Comment (qaphla) [reply src/parser/jobs/pld/modules/OGCDDowntime.tsx]: Should be resolved in current version --- I also added brief explanations for the GNB numbers.

File: src/parser/jobs/gnb/modules/OGCDDowntime.tsx (hunk 2 above)
Comment (qaphla) [reply src/parser/jobs/gnb/modules/OGCDDowntime.tsx]: Fixed in current version.

**Pull request merged**